drop-in/src/main/java/com/adyen/checkout/dropin/DropIn.kt

```diff
@@ -243,6 +243,11 @@ object DropIn {
     }
 
     private fun updateDefaultLogLevel(context: Context) {
-        Logger.updateDefaultLogLevel(BuildUtils.isDebugBuild(context))
+        val logLevel = if (BuildUtils.isDebugBuild(context)) {
+            AdyenLogLevel.DEBUG
+        } else {
+            AdyenLogLevel.NONE
+        }
+        AdyenLogger.setLogLevel(logLevel)
```

Comment:
@OscarSpruit this is now causing an issue where drop-in overrides the log level explicitly set by the merchant. Check out `Logger.isLogLevelInitialized` in the old code